buildtest/buildsystem/schemas/utils.py

```diff
@@ -25,9 +25,9 @@ def load_schema(path):
     if not os.path.exists(path):
         sys.exit("schema file %s does not exist." % path)
     with open(path, "r") as fd:
-        if path.endswith("json"):
+        if path.endswith(".json"):
             schema = json.loads(fd.read())
-        elif path.endswith("ml"):
+        elif path.endswith(".yaml") or path.endswith(".yml"):
```

Comment:
Nit: this would be more elegant with:

```python
if re.search("(yaml|yaml)^", path):
```
Generally when you have to do more than one if this, if that, it's good to try and look for a regular expression that can represent both.

Reply:
thats good suggestion. Regex are very powerful and this would be good example. I need to get more comfortable with regex, i have some understanding but practically haven't used it much in the codebase as you can see.